test/unit/domainPathRouter.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { DomainPath } from '../../src/domainPaths/domainPath'
+import { DomainPathRouter } from '../../src/domainPaths/domainPathRouter'
+
+// ----------------------------------------------------------------
+// --------------------------- setup ------------------------------
+const dp1 = new DomainPath('domainpath')
+const dp2 = new DomainPath('alt-domainpath')
+const dp3 = new DomainPath('third-domainpath')
+const dpRouter = new DomainPathRouter({
+  'url1': dp1,
+  'url2': dp2,
+  'url2/subdomain/': dp3,
+})
```

Comment:
It seems to also work when you provide actual urls, so instead of `url1` => `example.com`. But is there a reason why it's not using actual urls? 

Also if I test it with `http(s)://example.com` the tests fail. It's not a big deal just FYI

Reply:
any comments on this?

Reply:
I've updated the urls. Seemed to work with proper urls.